odoo/addons/base/ir/ir_model.py

```diff
@@ -912,9 +912,12 @@ def _add_manual_fields(self, model):
         fields_data = self._get_manual_field_data(model._name)
         for name, field_data in fields_data.items():
             if name not in model._fields and field_data['state'] == 'manual':
-                field = self._instanciate(field_data)
-                if field:
-                    model._add_field(name, field)
+                try:
+                    field = self._instanciate(field_data)
+                    if field:
+                        model._add_field(name, field)
+                except Exception:
+                    _logger.warning("Manual field '%s' : loading skipped", field_data['name'])
```

Comment:
How about using `_logger.exception` to get the traceback in the logs as well?

```suggestion
                    _logger.exception("Failed to load field '%s': skipped", field_data['name'])
```

Reply:
You should also log the model name.

Reply:
Thank you. model name has been added and logger.exception is indeed better.